collector/configuration/liverig.properties.md

```diff
@@ -68,7 +68,14 @@
 | `witsml_tree_event_period_seconds` <br> **This Property was removed on Liverig 2.22.0**                                                                                                                                                                                                                                                            |   `long`   | `600`                                | 2.17.0 |     s     |                                                                                                          |                          |
 | `wildcard_period_in_seconds` <br> Sets the Wildcard Period if the Request does not contain it.                                                                                                                                                                                                                                                     |   `int`    | `600`                                | 2.22.0 |     s     |                                                                                                          |                          |
-| `enable_revision`                                                                                                                                                                                                                                                                                                                                  | `boolean`  | `false`                              | 2.24.1 |           |                                                                                                          |                          |
+| `witsml_change_mechanism_feature` <br> Enable/Disable WITSML change mechanism. | `boolean` | `true` | 5.17.x |||||
+| `witsml_change_mechanism_delay_period_in_seconds` <br> Delay period of WITSML change mechanism. | `int` | `0` | 5.17.x |s||||
+| `witsml_change_mechanism_detection_period_in_seconds` <br> WITSML change detection period mechanism. | `int` | `10` | 5.17.x |s||||
+| `witsml_change_mechanism_initial_last_change_feature` <br> Enable/Disable WITSML initial last change mechanism. | `boolean` | `true` | 5.17.x |||||
+| `witsml_change_mechanism_late_channels_feature` <br> Enable/Disable WITSML late channel change mechanism. | `boolean` | `true` | 5.17.x |||||
+| `witsml_change_mechanism_object_growing_feature` <br> Enable/Disable WITSML object growing change mechanism. | `boolean` | `true` | 5.17.x |||||
+| `witsml_change_mechanism_initial_last_change_in_minutes` <br> WITSML last change in minutes. | `int` | `1` | 5.17.x | minutes ||||
```

Comment:
The default values are present in AppProperties.java like the following:
```java
    public static final boolean DEFAULT_WITSML_CHANGE_MECHANISM_FEATURE = false;
    public static final boolean DEFAULT_WITSML_CHANGE_MECHANISM_LATE_CHANNELS_FEATURE = false;
    public static final boolean DEFAULT_WITSML_CHANGE_MECHANISM_OBJECT_GROWING_FEATURE = false;
    public static final long DEFAULT_WITSML_CHANGE_MECHANISM_DETECTION_PERIOD_IN_SECONDS = 10;
    public static final boolean DEFAULT_WITSML_CHANGE_MECHANISM_INITIAL_LAST_CHANGE_FEATURE = true;
    public static final long DEFAULT_WITSML_CHANGE_MECHANISM_INITIAL_LAST_CHANGE_IN_MINUTES = 1440;
    public static final long DEFAULT_WITSML_CHANGE_DELAY_PERIOD_IN_SECONDS = 0;
```

please fix the documentation here to reflect the correct values.